src/we_internal.c

```diff
@@ -1311,6 +1311,21 @@ static const ECDSA_METHOD *we_ecdsa(void)
 #endif
 #endif /* WE_HAVE_ECDSA */
 
+#ifdef HAVE_FIPS
+static void fipsCb(int ok, int err, const char* hash)
+{
+    printf("in FIPS callback, ok = %d, err = %d\n", ok, err);
+    printf("message = %s\n", wc_GetErrorString(err));
+    printf("hash = %s\n", hash);
+
+    if (err == IN_CORE_FIPS_E) {
+        printf("In core integrity hash check failure. Copy above hash\n");
```

Comment:
I think this should have more markers to make it more prominent. like
```
printf("*******************************************\n");
```

Reply:
Added.